manuscript/algorithm.tex

```diff
@@ -0,0 +1,77 @@
+\section{ Demand-driven deployment models}
+To predict the market demand for nuclear fuel cycle simulators, various algorithms
+can be adopted. The algorithms can be divided into three major
+branches, Non-optimizing, Deterministic Optimizing, and Stochastic
+Optimizing. Each groups vary in complexity, which leads to a difference
+in computing time and accuracy. The groups and their examples are 
+explained in detail below. 
+
+\subsection{Non-Optimizing(NO)}
+
+Non optimizing algorithms predict future deployment schedules
+only based on the past datasets. These algorithms do not attempt
+to minimize the difference of the produced quantity to the demand,
```

Comment:
Break this into two sentences and flip: "In the case of demand-driven deployment, a core objective may be to minimize the discrepancy between the production and demand of a certain quantity. However, as their name suggests, non-optimizing algorithms do not attempt to minimize or maximize any such objective functions."

minimize or maximize any objective function. Thus the discrepancy between production and demand.)